src/levels.rs

```diff
@@ -190,7 +190,7 @@ pub const INTER_INTRA_NONE: InterIntraType = 0;
 /// which includes `&self` methods, including autogenerated ones like [`PartialEq::eq`].
 ///
 /// [`refmvs_block`]: crate::src::refmvs::refmvs_block
-#[derive(Clone, Copy, PartialEq, Eq, Default)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Default)]
```

Comment:
The previous format string was `y=%d,x=%d`, so the `Debug` (or `Display`) `impl` should at least preserve that.  That said, I don't think we should use a `Debug` `impl` for this at all, as `mv` is commonly formatted in other ways elsewhere, such as

```rust
            println!(
                "Post-dmv[{}/{},ref={}/{}|{}/{}]: r={}",
                b.mv()[0].y,
                b.mv()[0].x,
                r#ref.y,
                r#ref.x,
                mvstack[0].mv.mv[0].y,
                mvstack[0].mv.mv[0].x,
                ts.msac.rng,
            );
```

in #217.

Reply:
Yeah that's a fair point. I'll remove the `Debug` impl and revert the `println!` where I was using it to be the same as in the C version.